Vulnerability.Scanner.Api/Controllers/Domain/Reports.cs

```diff
@@ -64,6 +64,47 @@ public GroupModel PutGroup(string groupId, GroupScanRequest request)
             }
         }
 
+        public void RemoveGroup(string groupId)
+        {
+            var existing = _dataContext.Groups
+            .Where(x => x.GroupId == groupId);
+
+            if(existing != null) {
+                foreach (var removable in existing) {
+                    _dataContext.Groups.Remove(removable);
+                }
+
+                RemoveReportsFromGroup(groupId);
+
+                try { 
+                     _dataContext.SaveChanges();
+                }
+                catch (Exception e) {
+                    Console.WriteLine(e); 
+                }
+            }
+        }
+
+        public void RemoveReportsFromGroup(string groupId) {
+            var existing = _dataContext.Groups
+            .Where(x => x.GroupId == groupId)
+            .Include(x=> x.Reports)
+            .SelectMany(x=> x.Reports);
+
+            if(existing != null) {
```

Comment:
I'm seeing two styles with `if` statements; `if(....) {` and `if (...) {` should you just use _one_ style and not mixed ones?